docs/api/animation/AnimationAction.html

```diff
@@ -11,121 +11,363 @@
     <h1>[name]</h1>
 
     <div class="desc">
-    An AnimationAction schedules clip playback on specific objects.
+        AnimationActions schedule the performance of the animations which are stored in
+        [page:AnimationClip AnimationClips].<br /><br />
+        
+        Note: Most of AnimationAction's methods can be chained.<br /><br />
+        
+        For an overview over the different elements of the three.js animation system see the
+        "Animation System" article in the "Next Steps" section of the manual.
     </div>
-
+    
 
     <h2>Constructor</h2>
-
+    
 
     <h3>[name]( [page:AnimationMixer mixer], [page:AnimationClip clip], [page:Object3D localRoot] )</h3>
-
-
+    <div>       
+        [page:AnimationMixer mixer] - the *AnimationMixer* that is controlled by this action.<br />
+        [page:AnimationClip clip] - the *AnimationClip* that holds the animation data for this action.<br />
+        [page:Object3D localRoot] - the root object on which this action is performed.<br /><br />
+        
+        Note: Instead of calling this constructor directly you can instantiate an AnimationAction with 
```

Comment:
It would be best to explain the reasoning then

>Instead of calling this constructor directly you should instantiate an AnimationAction.clipAction since this method provides caching for better performance.